docs/nextflow.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Nextflow integration
+---
+
+You can use Wave directly from your Nextflow pipelines, with full support for private repositories, container freezing, and conda packages.
+
+This feature requires Nextflow `22.10.0` or later.
+
+You can use Wave can in any Nextflow pipeline by adding the following snippet to your `nextflow.config` file:
+
+```groovy
+wave {
+    enabled = true
+}
+
+tower {
+    accessToken = '<your_access_token>'
+}
+```
+
+:::tip
+The use of the Seqera access token is not mandatory. However, it grants the following additional capabilities:
+
+- Access to private repositories
+- More API requests than is permitted for anonymous users
+:::
+
+For the entire list of configuration options, see [Configuration options](#configuration-options).
+
+## Limitations
```

Comment:
Can we move limitations to the bottom? I'm not sure that it needs to be (almost) the first thing that the reader sees.